client/src/components/ProfileMenu/ProfileMenu.tsx

```diff
@@ -0,0 +1,33 @@
+import Grid from '@material-ui/core/Grid';
+import { NavLink } from 'react-router-dom';
+import { Typography } from '@material-ui/core';
+import useStyles from './useStyles';
+
+export default function ProfileMenu(): JSX.Element {
+  const classes = useStyles();
+
+  return (
+    <Typography>
```

Comment:
```suggestion
    <Box>
```

Maybe consider using a 'Box' component instead of a 'Typography' here for increased semantics